frontend/src/app/apps/[id]/page.tsx

```diff
@@ -1,104 +1,107 @@
-"use client";
+'use client';
 
-import React, { useEffect, useState } from "react";
-import styles from "@/styles/pages/application.module.css";
+import React, { useEffect, useState } from 'react';
+import styles from '@/styles/pages/application.module.css';
 import {
-  ApplicationExtended,
-  UserAppInformation,
-} from "@/app/api/applications/application.d";
-import ApplicationInformation from "@/components/ApplicationInformation/ApplicationInformation";
-import SkeletonLoader from "@/components/SkeletonLoader/SkeletonLoader";
-import BackLink from "@/components/BackLink/BackLink";
-import { useRouter } from "next/navigation";
+	ApplicationExtended,
+	UserAppInformation,
+} from '@/app/api/applications/application.d';
+import ApplicationInformation from '@/components/ApplicationInformation/ApplicationInformation';
+import SkeletonLoader from '@/components/SkeletonLoader/SkeletonLoader';
+import BackLink from '@/components/BackLink/BackLink';
+import { useRouter } from 'next/navigation';
 
 interface ApplicationProps {
-  params: {
-    id: string;
-  };
+	params: {
+		id: string;
+	};
 }
 
 const ApplicationPage = ({ params: { id } }: ApplicationProps) => {
-  const router = useRouter();
-  const [application, setApplication] = useState<ApplicationExtended>({
-    id: "",
-    name: "",
-    description: "",
-    owners: "",
-    teamName: "",
-    apiKey: "",
-    isValid: false,
-  });
-  const [loading, setLoading] = useState(true);
+	const router = useRouter();
+	const [application, setApplication] = useState<ApplicationExtended>({
+		id: '',
+		name: '',
+		description: '',
+		owners: '',
+		teamName: '',
+		apiKey: '',
+		isValid: false,
+	});
+	const [loading, setLoading] = useState(true);
 
-  async function fetchApplication(id: String): Promise<UserAppInformation> {
-    const res = await fetch(`/api/applications/${id}`, {
-      cache: "no-cache",
-    });
-    const application = await res.json();
-    setApplication(application);
-    setLoading(false);
-    return application;
-  }
+	async function fetchApplication(id: String): Promise<UserAppInformation> {
+		const res = await fetch(`/api/applications/${id}`, {
```

Comment:
Lets think about how we can minimize the amount of rewriting the api calls to common endpoints; possibly "common" pkg or something of fetch calls that simplifies future work.